### PR TITLE
Challenge 1 - Friend Referral 

### PR DESCRIPTION
**Pull request by rkyassine**

### CODE REVIEW

File: phases/phase-1/challenges/likes/Ping [hunk 1]
```diff
@@ -1 +1,2 @@
 EDCE438F71E8122C01321220A810A1BC4D18D0BEC52C01F82E876692C71C225F
+6FB64C92288133DE8B6A539906845F01665B5D7CE0EFF181D9CDDCDDC3EC6BE2 saiSunkari19 westaking
```
Comment (RiccardoM) [inline phases/phase-1/challenges/likes/Ping]:
```suggestion
```

I'm sorry but something but have gone wrong here. 

Please remove this line and run the following commands to fix this: 

```
echo 6FB64C92288133DE8B6A539906845F01665B5D7CE0EFF181D9CDDCDDC3EC6BE2 > ./phases/phase-1/challenges/likes/rkyassine
```

Comment (dafricash) [reply phases/phase-1/challenges/likes/Ping]: Hi, this folder is not mine its ping's one =>https://github.com/liangping.
For some reason it ended up in my pull request and also contains my transaction hash. 6FB64C92288133DE8B6A539906845F01665B5D7CE0EFF181D9CDDCDDC3EC6BE2
this is my PR
https://github.com/desmos-labs/primer/blob/927bbdf125de94c2a7ebc5493ffdc77f9c60d2af/phases/phase-1/challenges/likes/rkyassine

Comment (dafricash) [reply phases/phase-1/challenges/likes/Ping]: Hi, this folder is not mine its ping's one =>https://github.com/liangping.
For some reason it ended up in my pull request and also contains my transaction hash. 6FB64C92288133DE8B6A539906845F01665B5D7CE0EFF181D9CDDCDDC3EC6BE2
this is my PR
https://github.com/desmos-labs/primer/blob/927bbdf125de94c2a7ebc5493ffdc77f9c60d2af/phases/phase-1/challenges/likes/rkyassine

Comment (rkyassine) [reply phases/phase-1/challenges/likes/Ping]: Hi, this folder is not mine its ping's one =>https://github.com/liangping.
For some reason it ended up in my pull request and also contains my transaction hash. 6FB64C92288133DE8B6A539906845F01665B5D7CE0EFF181D9CDDCDDC3EC6BE2
this is my PR
https://github.com/desmos-labs/primer/blob/927bbdf125de94c2a7ebc5493ffdc77f9c60d2af/phases/phase-1/challenges/likes/rkyassine

Comment (RiccardoM) [reply phases/phase-1/challenges/likes/Ping]: Can you please commit my suggestion so this will be fixed? Or allow me to push here. Thanks

**Comment by RiccardoM** [comment]
@rkyassine Please don't delete the file from Ping

Please execute to restore the file as it was originally, otherwise I won't be able to merge this. 

```
echo EDCE438F71E8122C01321220A810A1BC4D18D0BEC52C01F82E876692C71C225F > ./phases/phase-1/challenges/likes/Ping
```

**Comment by rkyassine** [comment]
sorry for that, i restored it 
https://github.com/desmos-labs/primer/pull/48/commits/9265c8f1fc838459e26880dee8d818383373e448

**Comment by RiccardoM** [comment]
> sorry for that, i restored it
> [9265c8f](https://github.com/desmos-labs/primer/commit/9265c8f1fc838459e26880dee8d818383373e448)

Thanks, and sorry for the inconvinience

**Pull request merged**